h/static/scripts/app.coffee

```diff
@@ -32,6 +32,14 @@ resolve =
         action = message.options.action
         annotations = message.payload
         return unless annotations?.length
+
+        # Discard annotations that aren't from the currently focused group.
+        # FIXME: Have the server only send us annotations from the focused
+        # group in the first place.
```

Comment:
As a separate task, I think it would be very useful to display counts of annotations across different groups in the groups dropdown, so even if the server sends only the annotation payloads for the focused group, having facet counts from ES for the other groups would be very useful.